srcpkgs/gammaray/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gammaray'
+pkgname=gammaray
+version=2.11.1
+revision=1
+build_style=cmake
+configure_args="-DGAMMARAY_INSTALL_QT_LAYOUT=true -DECM_MKSPECS_INSTALL_DIR=/usr/lib/qt5/mkspecs/modules -DPLUGIN_INSTALL_DIR=/usr/lib/qt5/plugins/gammaray"
+makedepends="qt5-devel"
+short_desc="Tool to poke around in a Qt-application"
+maintainer="toluschr <toluschr@protonmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/KDAB/GammaRay"
+distfiles="https://github.com/KDAB/GammaRay/releases/download/v${version}/${pkgname}-${version}.tar.gz"
+checksum=87a1d72ad1ad6d1a0156c54a85b0976ab38c6a64136458ca7c4ee491566d25d0
+
+LDFLAGS+="-Wl,-no-fatal-warnings"
+
+pre_build() {
+	sed -e 's|plugins/gammaray|lib/qt5/plugins/gammaray|' -i CMakeLists.txt
+}
+
+gammaray-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision} ${makedepends}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/cmake
+		vmove usr/lib/qt5/mkspecs
+		cd "${DESTDIR}"
+		for f in usr/lib/*.so; do [ -L "${f}" ] && vmove "${f}"; done
```

Comment:
Does it ship any `.so` files that aren't symlinks? Ideally this line should be `vmove "usr/lib/*.so"`.

And I'm not entirely sure that `cd` makes everything work normally, you should probably check the package contents afterwards.

Reply:
Yes, this is necessary

Reply:
> And I'm not entirely sure that `cd` makes everything work normally

I used cd to make the glob expression use the relative path

Reply:
Seems like `vmove` is smart enough to deal with it. Are there too many of these files, and some specific reason for them not being versioned?

Reply:
I don't know why the shared libraries are not versioned. I believe they act as plugins.

Reply:
I see. If you could leave a comment there, then. Like "unversioned sonames should go in main package"